docker/pg/initdb.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+/usr/lib/postgresql/10/bin/pg_ctl -D $PGDATA -w start
+pgbench -i -s 10 postgres
+pg_dumpall -f /tmp/dump1
+wal-g backup-push $PGDATA
+pkill -9 postgres
```

Comment:
Such statements can scare the hell out of DBA's...
but it leaves less logs than pg_ctl stop --immediate, let's leave it

Reply:
It's a disaster recovery test. I've just considered "pkill -9" to be one.